### PR TITLE
fix/cys ui feedback 12 sep

### PR DESCRIPTION
**Pull request by rjchow**
- added utility to calculate background colours
- ai wizard visual tweaks
- changed scaleY to opacity instead to mitigate the child scaling issue
- fixed padding and double scrollbars

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Addressed background colours not adhering to colour scheme:
Before:
image
After:
image

Addressed typo and text padding
Before:
image
After:
image

Addressed copy change and text padding
Before:
image
After:
image

Addressed spotlight overlay opacity
Before:
image

After:
image

Addressed stretched tooltip text, padding on the right and double scrollbars
Before:
image
After:
image

(note that the stretched tooltip was addressed by removing the interactive vertical enlargement of the handle, and animating the opacity instead)

Not addressed:
- Tourkit frame arrow placement not centered
- size of preview frame scrollbar
- loading time required to move from page to page
- default colour palette
- unset active state on color palette when user changes colors
- default font choice

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

This PR addressed some visual tweaks for the customize your store feature that is still behind the `customize-store` feature flag and does not need QA review before merging

1. Use WP 6.3 
1. Install the `WooCommerce Beta Tester` plugin
1. Go to /wp-admin/tools.php?page=woocommerce-admin-test-helper and enable `customize-store` feature flag
1. Go to `WooCommerce > Home`
1. Click on `Customize your store` task 
1. Click on `Assembler Hub` button and test around the changes mentioned above
1. Go back to customize your store intro screen and click on `Design with AI`, and test around the changes mentioned above

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss [hunk 1]
```diff
@@ -374,13 +375,13 @@
...
 			gap: 10px;
-			background: #d7defb;
+			background: var(--wp-admin-theme-color-background-25);
 			left: 5px !important;
 			border-radius: 4px;
 			height: 20px;
 
 			.components-popover__content {
-				color: #1d35b4;
+				color: var(--wp-admin-theme-color);
```
Comment (chihsuan) [inline plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss]:
```suggestion
				color: var(--wp-admin-theme-color-darker-20);
```
Mrk6SERPZ4KrFHSjM0a8TK-fi-2786%3A280145
Screenshot 2023-09-15 at 10 28 53

File: plugins/woocommerce-admin/client/customize-store/style.scss [hunk 2]
```diff
@@ -77,7 +82,7 @@ body.woocommerce-customize-store.js.is-fullscreen-mode {
 		padding: 12px;
 		width: 404px;
 		border-radius: 2px;
-		background: var(--gutenberg-transparent-blueberry, rgba(56, 88, 233, 0.04));
+		background: var(--wp-admin-theme-color-background-04, rgba(168, 168, 170, 0.301));
```
Comment (chihsuan) [inline plugins/woocommerce-admin/client/customize-store/style.scss:85]: Should we use `rgba(56, 88, 233, 0.04)` for fallback? Or can you point out where `rgba(168, 168, 170, 0.301)` comes from? 😅

File: plugins/woocommerce-admin/client/utils/derive-wp-admin-background-colours.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	return newHex;
+}
+
+/**
+ * wp-admin theme colour only include the main colour,
+ * but in some applications we want to derive a complementary
+ * background colour that's some percentage lighter than the
+ * wp-admin theme colour. This is not doable in CSS as it involves
+ * breaking down the hex colour code and then running calculations on it.
+ * As of writing, CSS calc can only operate on individual numbers
```
Comment (chihsuan) [inline plugins/woocommerce-admin/client/utils/derive-wp-admin-background-colours.ts:31]: 👍

**Review by chihsuan — APPROVED** [review]
Just left two suggestions on colors, others tested well. Thanks for fixing this! Pre-approving! 👍 

> (note that the stretched tooltip was addressed by removing the interactive vertical enlargement of the handle, and animating the opacity instead)

Nice approach. 😄

File: plugins/woocommerce-admin/client/customize-store/style.scss (hunk 2 above)
Comment (rjchow) [reply plugins/woocommerce-admin/client/customize-store/style.scss:85]: I just picked a grey that would work with any theme colour 🫣

Comment (chihsuan) [reply plugins/woocommerce-admin/client/customize-store/style.scss:85]: Ok, make sense 😆

File: plugins/woocommerce-admin/client/customize-store/style.scss [hunk 4]
```diff
@@ -19,6 +19,11 @@
 
 body.woocommerce-customize-store.js.is-fullscreen-mode {
 	margin-top: 0 !important;
+	height: unset;
+
+	& > div.tour-kit.woocommerce-tour-kit > div > div.tour-kit-spotlight.is-visible {
+		outline: 99999px solid rgba(0, 0, 0, 0.15);
```
Comment (ilyasfoo) [inline plugins/woocommerce-admin/client/customize-store/style.scss:25]: Clever! I've never seen an outline used overlay outside of its boundaries before. I wonder if this screws up some obscure browsers with questionable rendering. Not a concern though!

**Review by ilyasfoo — APPROVED** [review]
Tested well, thanks! 🚢

**Pull request merged**